Mathlib/Data/Polynomial/Module.lean

```diff
@@ -14,17 +14,58 @@ In this file, we define the polynomial module for an `R`-module `M`, i.e. the `R
 
 This is defined as a type alias `PolynomialModule R M := ℕ →₀ M`, since there might be different
 module structures on `ℕ →₀ M` of interest. See the docstring of `PolynomialModule` for details.
```

Comment:
Can you add a couple of lines here in the module header about `AEval` also?

Reply:
Thanks for pointing that out.